pvdeg/standards.py

```diff
@@ -139,36 +256,178 @@ def standoff(
     elif weather_df is None:
         weather_df, meta = weather.get(**weather_kwarg)
 
-    if module_type == "glass_polymer":
-        conf_0 = "insulated_back_glass_polymer"
-        conf_inf = "open_rack_glass_polymer"
-    elif module_type == "glass_glass":
-        conf_0 = "close_mount_glass_glass"
-        conf_inf = "open_rack_glass_glass"
-
     solar_position = spectral.solar_position(weather_df, meta)
     poa = spectral.poa_irradiance(
-        weather_df,
-        meta,
+        weather_df=weather_df,
+        meta=meta,
         sol_position=solar_position,
         tilt=tilt,
         azimuth=azimuth,
-        sky_model=sky_model,
-    )
-    T_0 = temperature.module(
-        weather_df, meta, poa, temp_model, conf_0, wind_speed_factor
-    )
-    T_inf = temperature.module(
-        weather_df, meta, poa, temp_model, conf_inf, wind_speed_factor
-    )
-    x, T98_0, T98_inf = eff_gap(T_0, T_inf, level=level, T98=T98, x_0=x_0)
-
-    res = {"x": x, "T98_0": T98_0, "T98_inf": T98_inf}
+        sky_model=sky_model, )
+    T_0 = temperature.cell(
+        weather_df=weather_df, meta=meta, poa=poa, temp_model=temp_model, conf=conf_0, wind_factor=wind_factor )
+    T98_0 = T_0.quantile(q=0.98, interpolation="linear")
+    T_inf = temperature.cell(
+        weather_df=weather_df, meta=meta, poa=poa, temp_model=temp_model, conf=conf_inf, wind_factor=wind_factor )
+    T98_inf = T_inf.quantile(q=0.98, interpolation="linear")
 
+    try:
+        x = -x_0 * np.log(1 - (T98_0 - T98) / (T98_0 - T98_inf))
+    except RuntimeWarning as e:
+        x = np.nan # results if the specified T₉₈ is cooler than an open_rack temperature 
+    if x<0: 
+        x=0 
+
+    res = {"x": x, "T98_0": T98_0, "T98_inf": T98_inf, "T98": T98}
     df_res = pd.DataFrame.from_dict(res, orient="index").T
 
     return df_res
 
+def Interpret_Standoff(
```

Comment:
please check out python naming conventions: https://peps.python.org/pep-0008/#function-and-variable-names

"Function names should be lowercase, with words separated by underscores as necessary to improve readability."

Reply:
I removed the capitals. This required the standards.py and the standards Jupyter notebook to be changed.

However, we are still using T98_estimate as a name.

Reply:
thanks you! yes, capitalized T for temperature is still left over from the initial functions. I'll add it to our list of issues that should be updated.